webdriver/tests/bidi/storage/set_cookie/invalid.py

```diff
@@ -54,41 +54,41 @@ async def test_cookie_name_invalid_type(bidi_session, test_page, domain_value, n
         "no_leading_forward_slash"
     ]
 )
-async def test_cookie_path_invalid_value(bidi_session, test_page, domain_value, path):
+async def test_cookie_path_invalid_value(set_cookie, test_page, domain_value, path):
```

Comment:
Same as for the `name` Firefox API doesn't fail to set cookies with these paths

Reply:
ack. Let's keep it out of scope of this PR

Reply:
@lutien mind creating a PR removing invalid cases where Firefox does not fail?

Reply:
Sure, I'll do it